docs/modules/mobile/reconnection-logic.md

```diff
@@ -0,0 +1,70 @@
+<%(TOC)%>
+
+# Reconnection logic
+
+Mobile applications present new challenges in terms of handling client connections. A client may disconnect due to an unstable connection or when the application is put into a background state by the OS. We want to dive a bit deeper into the causes of a disconnect and how you can handle it in your game.
+
+## Why clients disconnect
+
+### Unstable connection 
+
+When creating a mobile application, you need to ensure that [client-workers]({{urlRoot}}/reference/glossary#client-worker) are able to run the game even with an unstable connection. To achieve this, you can implement connection handling to ensure that those client-workers are able to reconnect.
+
+If a client-worker's connection is too unstable, they might not be able to connect to your game or might disconnect at any time. You should provide a connection flow that allows them to gracefully disconnect and handle reconnection when they have a more stable connection.
+
+### Pausing of applications
+
+When pausing an application or putting it into the background, there are two possible scenarios that can cause a disconnect:
+
+#### 1. The device OS closes the application.
+
+If the application has been closed, the user has to restart the application. The user will be brought back to the start screen the next time the application is opened and from there on, the application can connect and load any data necessary.
+
+#### 2. The client stops sending data. 
+
+This scenario is a bit more tricky. The application is still alive, however the OS may decide to not run the game or send any data while the application is in the background. [SpatialOS]({{urlRoot}}/reference/glossary#spatialos-runtime) will close the connection if it doesn't receive any messages from a client-worker for a period of time. You need to add additional reconnection logic to your game to handle this scenario. 
```

Comment:
```suggestion
This scenario is a bit more tricky. The application is still alive, however the OS may decide to not run the game or send any data while the application is in the background. [SpatialOS]({{urlRoot}}/reference/glossary#spatialos-runtime) will close the connection if it doesn't receive any messages from a client-worker for a period of time. 

When the application is brought back to the foreground, it will act as if the connection was dropped. You need to add additional reconnection logic to your game to handle this scenario. 
```

This is handled basically the same as the Unstable Connection section right? Do we want to reference that?